make/istio/run

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/bin/common/versions.sh"
+source "${GIT_ROOT}/make/include/defaults"
+
+# Download and install the Istio release.
+istio_path="${GIT_ROOT}/.istio/${ISTIO_VERSION}"
+if [ ! -d "$istio_path" ]; then
+  mkdir -p "${istio_path}"
+  curl -L "https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-linux.tar.gz" \
+    | tar zx -C "${istio_path}" --strip-components=1
+fi
+
+path_with_istio="PATH=\"${istio_path}/bin:\${PATH}\""
+export_path_comment="# Add Istio binaries to PATH."
+if ! grep --silent "${export_path_comment}" ~/.profile 2> /dev/null; then
```

Comment:
Hmm, I am not sure I'm entirely comfortable with this modification of user data.
You can actually run `make/run`, etc. on your workstation (pointing at a kube), and things will work. So modifying `~/.profile` here, instead of when building vagrant (or at least, `vagrant up`), seems icky.